src/core/createLogController.js

```diff
@@ -24,8 +24,9 @@ export default ({
   const parsedQueryString = queryString.parse(locationSearch);
   const storage = createNamespacedStorage(`instance.${instanceNamespace}.`);
 
-  let debugEnabled = storage.session.getItem("debug") === "true";
-  let debugEnabledWritableFromConfig = true;
+  const debugSessionValue = storage.session.getItem("debug");
+  let debugEnabled = debugSessionValue === "true";
+  let debugEnabledWritableFromConfig = debugSessionValue === null;
```

Comment:
I could have also done `!debugSessionValue` but I wanted to be clear that I was trying to match `null` and not erroneously trying to match `"false"`.